### ISSUE TITLE
Add Bootstrap Method

### ISSUE THREAD

**Issue by ddmorgan** [uw-cmg/MAST-ML#164]
The basic bootstrap methods give error bars for any data being predicted.  Worth putting in as a standard tool to get an error bar.  See  https://en.wikipedia.org/wiki/Bootstrapping_(statistics) for details.  Likely this is in psikitlearn.

**Comment by qpwo** [comment]
Sounds like good idea... I will look into implementation difficulty on Monday. Maybe we can use http://scikit-learn.org/stable/modules/generated/sklearn.utils.resample.html

**Comment by qpwo** [comment]
We have a very preliminary version of this working now. I think within a couple weeks we will have some standard way to add error bars

**Comment by qpwo** [comment]
@ddmorgan Right now we use `sklearn.model_selection.RepeatedKFold` to get stdev/stderr of predictions across folds, or with classification to see how often a point is put in each class. Does bootstrapping have an advantage over our current method?

**Comment by ddmorgan** [comment]
I think bootstrapping is different.  It is a rigorous statistical method that involves a very specific kind of sampling (which is not k-fold CV) and gives a certain kind of information on error.  I don't fully understand what that error means, but it is not the same as CV error I don't think.

**Comment by ddmorgan** [comment]
Note that bootstrap is effectively what is used in random forest so those error bars may be related to bootstrap-based error bars.

**Comment by rjacobs914** [comment]
Bootstrapping data splitter added in latest branch
